### PR TITLE
Zero Tier Information Tables

### PR DESCRIPTION
**Pull request by directionless**
Add informational reporting tables for zero tier one

### CODE REVIEW

File: pkg/osquery/table/zerotier_windows.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// +build windows
+
+package table
+
+import "path"
+
+// zerotierCli returns the path to the CLI executable.
+func zerotierCli(args ...string) []string {
+	cmd := []string{
+		path.Join("C:", "ProgramData", "ZeroTier", "One", "zerotier-one_x64.exe"),
```
Comment (blaedj) [inline pkg/osquery/table/zerotier_windows.go:10]: It's probably not a huge deal, but such a hardcoded path makes me a little uneasy. 

We can likely change it easily enough, but the turnaround time for a launcher change is fairly long compared to changes in a query.. The obvious alternative of supplying an exec path as a query constraint doesn't seem any better to me though.

**Review by blaedj — APPROVED** [review]
lgtm

File: pkg/osquery/table/zerotier_windows.go (hunk 1 above)
Comment (terracatta) [reply pkg/osquery/table/zerotier_windows.go:10]: Can we pass in some environment variables here? Like instead of `C:\` we could be using `SYSTEMROOT`

Comment (directionless) [reply pkg/osquery/table/zerotier_windows.go]: I updated this to use `os.GetEnv("SYSTEMROOT")` but until we see more of these, I don't think we there's a good way to be more clever .

Best case, we could hardcode a bunch of places to check. But without knowing more, it's needless boilerplate.

**Pull request merged**